otp4gb/logging.py

```diff
@@ -8,12 +14,12 @@
     "%(asctime)s.%(msecs)03d:%(levelname)s:%(processName)s:%(module)s:%(message)s"
```

Comment:
I take it this is also no longer needed (as unused) - now taken care of by `initialise_logger` function?

Reply:
Yes that's correct, although I have added an issue (#12) to remove the `initialise_logger` function from here and use `caf.toolkit.LogHelper` to make logging more consistent and simpler